src/viam/sdk/services/private/mlmodel_server.cpp

```diff
@@ -12,12 +12,11 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#include <viam/sdk/services/private/mlmodel_server.hpp>
-
 #include <viam/sdk/common/private/service_helper.hpp>
 #include <viam/sdk/rpc/server.hpp>
 #include <viam/sdk/services/mlmodel.hpp>
 #include <viam/sdk/services/private/mlmodel.hpp>
+#include <viam/sdk/services/private/mlmodel_server.hpp>
```

Comment:
It looks like maybe your editor got a little aggressive in its formatting opinions, here and in other files. Can you put back the header ordering? It actually is intentional the way it is, with the associated header coming first in its own group (helps ensure the header is standalone).

Reply:
undid the bad linting

Reply:
This one should go back up as well.

Reply:
done